packages/editor/src/components/document-outline/index.js

```diff
@@ -48,7 +48,7 @@ const multipleH1Headings = [
  *
  * @return {Array} An array of heading blocks enhanced with the properties described above.
  */
-const computeOutlineHeadings = ( blocks = [], path = [] ) => {
+export const computeOutlineHeadings = ( blocks = [], path = [] ) => {
```

Comment:
It no longer should be exported:
```suggestion
const computeOutlineHeadings = ( blocks = [], path = [] ) => {
```